lib/main.ml

```diff
@@ -87,8 +87,8 @@ let file =
     ~doc:"Configuration file for Mirage. If not specified, the current directory will be scanned. \
           If one file named $(b,config.ml) is found, that file will be used. If no files \
           or multiple configuration files are found, this will result in an error unless one \
-          is explicitly specified on the command line." [] in
-  Arg.(value & pos 0 (some string) None & doc)
+          is explicitly specified on the command line." ["f"; "file"] in
```

Comment:
do you really need this? It will break `mirage-skeleton` and needs some updates to `mirage-www` (both are fine if they are really needed, though).